Dockerfile-buster-slim.template

```diff
@@ -21,11 +21,12 @@ EXPOSE 4465
 EXPOSE 4466
 
 RUN apt-get update && apt-get upgrade -y && \
-	apt-get install -y openssl && \
-	apt-get clean && \
-	rm -rf /var/lib/apt/lists/* 
+    apt-get install -y openssl && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
 
-RUN useradd --system idsvr
+RUN groupadd --system --gid 999 idsvr && \
+    useradd --system --gid idsvr --uid 999 --shell /bin/bash --create-home idsvr
```

Comment:
Why opt to create a `/home/idsvr` directory from the skeleton template? I could understand `--home-dir /opt/idsvr` since that's where all the server files live.

Reply:
Home directories and the software the user is running are two separate things. But we can debate since a home directory wasn't being created before, maybe I shouldn't be introducing one. Though that's a notion I disagree with. Right now, if you run `cd ~`, it will say `cd: can't cd to /home/idsvr`.

Reply:
I think it's fine to make `cd ~` work. IMHO, I would expect it to change to `/opt/idsvr` since that's where all the action is. System accounts don't usually have directories in `/home`.

Reply:
IMHO, doing that goes against the Linux Filesystem Hierarchy Standard. But I'll leave the ultimate call about that to Curity folks.

Reply:
Another thing to think about is the shell profile stuff. With /opt/idsvr as the home, this stuff would have to be placed in /opt/idsvr too, which is kinda weird since then there'd be a mix of home content and application content in that directory.

Here's what is created when creating a new home that would be combined with /opt/idsvr content:
```
$ ls -a ~
.  ..  .bash_logout  .bashrc  .profile
```

Reply:
Those skeleton files get copied when you use the `--create-home` flag but _not_ when you use the `--home-dir` flag. I don't recommend copying those files.

Reply:
That's true, but I thought their presence would be desirable.